website/docs/docs/build/documentation.md

```diff
@@ -128,9 +129,11 @@ models:
 
 In the resulting documentation, `'{{ doc("table_events") }}'` will be expanded to the markdown defined in the `table_events` docs block.
 
-## Setting a custom overview
 
-The "overview" shown in the documentation website can be overridden by supplying your own docs block called `__overview__`. By default, dbt supplies an overview with helpful information about the docs site itself. Depending on your needs, it may be a good idea to override this docs block with specific information about your company style guide, links to reports, or information about who to contact for help. To override the default overview, create a docs block that looks like this:
+### Setting a custom overview
+*Currently available for the legacy dbt Docs only.*
```

Comment:
adding this clarification. opted for this small italics rather than separating this with a header since e some of these configs will be available in explorer.